.github/workflows/test.yml

```diff
@@ -85,12 +85,37 @@ jobs:
       - id: files
         name: Run the action
         uses: ./
+        with:
+          glob-filter: '*'
```

Comment:
I had skipped that one 🙈 I have fix it on main 😉

Reply:
Oh no! Sorry!